internal/discord/temp.go

```diff
@@ -0,0 +1,102 @@
+package discord
+
+import (
+	"fmt"
+	"slices"
+	"time"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+// Functions/methods that are meant to replace the existing non-temp versions during refactoring
```

Comment:
let's attach the issue number ([#19](https://github.com/naurffxiv/moddingway/issues/19))